### PR TITLE
Add ability to search through publishes.

### PR DESCRIPTION
**Pull request by Ruk33**
## Fixes

Issue Number: https://github.com/lbryio/lbry-desktop/issues/3139

## What is the current behavior?

Users can't search through their publishes.

## What is the new behavior?

Users can search through their publishes:

image

image

image

## PR Checklist

<details><summary>Toggle...</summary>

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I added a line describing my change to CHANGELOG.md
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

</details>

### CODE REVIEW

File: ui/page/fileListPublished/view.jsx [hunk 1]
```diff
@@ -42,12 +58,58 @@ function FileListPublished(props: Props) {
...
-    if (paramsString && fetchClaimListMine) {
-      const params = JSON.parse(paramsString);
-      fetchClaimListMine(params.page, params.page_size, true, filterBy.split(','));
+    if (!disableHistory) {
+      const params = new URLSearchParams(search);
+      params.set(PAGINATE_PARAM, '1');
+      history.push('?' + params.toString());
     }
-  }, [paramsString, filterBy, fetchClaimListMine]);
+  }, [filteredClaims]);
```
Comment (Ruk33) [inline ui/page/fileListPublished/view.jsx:108]: @jessopb I'm not entirely happy with this but I couldn't think of a better way to work around this issue.

Comment (jessopb) [reply ui/page/fileListPublished/view.jsx:108]: Seems to work really well. I don't think you need "disableHistory" - that mostly doesn't exist except if it's passed to <Paginate as a prop, and only on one page.

**Comment by jessopb** [comment]
I think this works well. 2 things, 
1) During loading, let's not say "No results".
2) It would be good to debounce the filtering while typing?
3) Maybe make Reposts searchable or disabled?

**Comment by jessopb** [comment]
I think we should put the search term in the params too, so when you navigate forward, then back again, you have both page and search term.

File: ui/page/fileListPublished/view.jsx (hunk 1 above)
Comment (Ruk33) [reply ui/page/fileListPublished/view.jsx]: Updated. Thanks @jessopb for the comment.

**Pull request merged**